consensus/core/src/core.rs

```diff
@@ -367,6 +367,19 @@ impl Core {
         self.commit_observer.handle_commit(committed_leaders)
     }
 
+    pub(crate) fn get_highest_accepted_rounds(&self) -> Vec<Round> {
+        let blocks = self
```

Comment:
Why not have synchronizer call this method from DagState directly?

Reply:
That would require injecting the DagState which I wanted to avoid doing here since we already have access to the core thread handle. Do you think it will be an issue?